sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/InternalRow.scala

```diff
@@ -32,32 +32,36 @@ abstract class InternalRow extends Serializable {
 
   def get(ordinal: Int): Any
 
-  def getAs[T](ordinal: Int): T = get(ordinal).asInstanceOf[T]
+  def genericGet(ordinal: Int): Any = get(ordinal, null)
```

Comment:
This is created as a special marker. For places that we currently use get without a data type that will be gone but I don't have time to delete yet (or the ones that don't matter), I will replace them with this genericGet.